src/apps/integrations/oneup_health/service/domain.py

```diff
@@ -9,3 +9,4 @@ class EHRData(InternalModel):
     healthcare_provider_id: str
     resources: list[dict]
     unique_id: uuid.UUID
+    activity_id: uuid.UUID
```

Comment:
Technically `unique_id` is no longer "unique" because it's essential for `activity_id` to also be included to make each row unique. Should we just rename `unique_id` to `submit_id` to be clear about what value it represents?
```suggestion
    submit_id: uuid.UUID
    activity_id: uuid.UUID
```